### PR TITLE
{lib}[GCCcore/7.3.0] libwebp v1.0.2

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)
Depends on ~~#7663~~

### CODE REVIEW

File: easybuild/easyconfigs/l/libwebp/libwebp-1.0.2-GCCcore-7.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+dependencies = [
+    ('libjpeg-turbo', '2.0.0'),
+    ('libpng', '1.6.34'),
+    ('LibTIFF', '4.0.9'),
+    ('giflib', '5.1.4'),
+]
+
+sanity_check_paths = {
+    'files': [],
```
Comment (boegel) [inline easybuild/easyconfigs/l/libwebp/libwebp-1.0.2-GCCcore-7.3.0.eb]: @akesandgren The existing `libwebp` easyconfig has a significantly more specific `sanity_check_paths`...

Comment (akesandgren) [reply easybuild/easyconfigs/l/libwebp/libwebp-1.0.2-GCCcore-7.3.0.eb]: Hmm, I apparently missed that there was one.
I'll update this one.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/8cddce00e94675fde88840e443667fcf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3134.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/14c35145c802931911dd77515797f049 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2402.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/03896a64086894be4920c09e7fcc7bfd for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2077.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/24a1eff7b8ff40d2d18f0b7361822cb9 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**